### PR TITLE
Modal popup 2 Pull request

### PR DESCRIPTION
**Pull request by adamteddychang**
Features implemented:
- project cards created with javascript
- mobile and desktop version modal popup 
- modal content created with javascript

### CODE REVIEW

File: modal.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  popupimg.src = dataobj.mainimage;
+  livebutton.href = dataobj.livelink;
+  sourcebutton.href = dataobj.sourcelink;
+
+  document.querySelector('#modal').style.display = 'block';
+}
+
+document.querySelectorAll('.st-see-btn_2').onclick = projectdetails(projectsarr);
+document.querySelectorAll('.closebtn').onclick = closepopup();
+document.querySelector('#st-see-btn').onclick = project0pop();
```
Comment (DanteAlonsoHT) [inline modal.js:134]: - [ ] Remember to follow the Figma design to make more impressive your portfolio.
- [ ] Fix the "X" close logo to close the popup, use the right logo according to the Figma design.
- [ ] Fix the background outside your popup, it should be blurred.
- [ ] [OPTIONAL] Fix the alignment between your text and logo in your "See demo" and "See repository" buttons.

**Your design:**

image

**Figma design, design expected:**

image

File: modal.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  document.querySelector('#other-proj-contain').innerHTML
+  += ` 
+  <div class="bk-img-proj">
+      <h2 class="bk-img-title">${project.title}</h2>
+      <p class="bk-img-p">${project.description}</p>
+      <ul class="lang-contain_2">
+        ${technologieslang}
+       </ul>
+      <button onclick='projectdetails(${index})' class="st-see-btn_2" type="button">See Project</button>
+  </div>
```
Comment (DanteAlonsoHT) [inline modal.js:87]: - [ ] When I decrease the width size of your popup, the x-scrollbar appears, please remove it.

image

Comment (adamteddychang) [reply modal.js:87]: I have removed the scrollbar completely. thank you!

File: modal.js (hunk 1 above)
Comment (rloterh) [reply modal.js]: This issue is well resolved! ✔️

Comment (adamteddychang) [reply modal.js]: Thank you Robert!

File: modal.js (hunk 2 above)
Comment (rloterh) [reply modal.js:87]: This issue is well resolved! ✔️

Comment (adamteddychang) [reply modal.js:87]: Thank you Robert!

File: modal.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
+// create the variables
+
+// Links for modal buttons
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply modal.js:3]: Thank you for the suggestions Robert I will do that

File: index.html [hunk 4]
```diff
@@ -219,8 +164,39 @@ <h4 class="form-title">I'm always interested in hearing about new projects,
...
+            <img class="popimg" src="#" alt="popupimg">
+            <div class="p-btn">
+            <p class="popupdescription"></p>
+            <div class="btnsdiv">
+            <a id="livebtn" class="st-see-btn_3 livebtn" href="">See live <img src="Images/share.png" alt="" id="shareimg"></a>
+            <a id="srcbtn" class="st-see-btn_3 livebtn" href="">See source <img src="Images/whitegithub.png" alt="" id="sourceimg"></a>
+            </div>
+            </div>
+    
+</div>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:193]: Sounds good to me! Thank you Robert for the suggestions!

**Pull request merged**